docs/security_vulnerability_disclosure/vulnerability_definition.md

```diff
@@ -0,0 +1,70 @@
+previous:
+  url: target_audience
+  title: Target Audience
+next:
+  url: user_tooling
+  title: User Tooling
+---
+
+Determining whether an issue qualifies as a security vulnerability requiring
+special handling or if it's merely a bug involves a nuanced evaluation process.
+Here's how a library maintainer can make this determination:
+
+1. **Understanding Security Vulnerabilities**: The maintainer should have a
+   clear understanding of what constitutes a security vulnerability. Common
+   examples include authentication bypass, injection attacks, sensitive data
+   exposure, and denial-of-service vulnerabilities. Familiarity with common
+   security issues helps in identifying potential vulnerabilities more
+   accurately.
+
+2. **Assessing Impact**: The maintainer should assess the potential impact of
+   the reported issue on the library's users and systems. Questions to consider
+   include:
+   - Could this issue lead to unauthorized access to sensitive data?
+   - Does it have the potential to compromise system integrity or availability?
+   - Can it be exploited to execute arbitrary code or perform actions beyond the
+     user's privileges?
+
+1. **Reviewing Context and Scope**: Understanding the context in which the issue
+   occurs and its scope is crucial. The maintainer should consider factors such
+   as:
+   - Whether the issue affects a critical component of the library or its core
+     functionality.
+   - If the issue has broader implications beyond the immediate codebase, such
+     as affecting dependent applications or integrations.
+   - Whether the issue is specific to certain configurations, environments, or
+     usage patterns.
+
+1. **Examining Attack Vectors**: Evaluating the potential attack vectors
+   associated with the reported issue provides insight into its severity.
+   Maintainers should consider:
+   - How easily an attacker can exploit the issue, including prerequisites and
+     required access levels.
+   - Whether the issue can be exploited remotely or requires local access.
+   - If exploitation requires user interaction or can be automated.
+
+2. **Considering Mitigation Complexity**: Assessing the complexity of mitigating
+   the issue helps determine the level of urgency and special handling required.
+   Factors to consider include:
+   - Whether a straightforward fix or workaround is available.
+   - If addressing the issue involves significant code changes or architectural
+     modifications.
+   - Whether mitigating the issue requires coordination with upstream
+     dependencies or third-party components.
+
+3. **Consulting Security Experts**: In complex cases or when uncertain, seeking
+   advice from security experts or the broader community can provide valuable
+   insights. Security professionals can offer guidance on threat modeling, risk
+   assessment, and appropriate mitigation strategies.
+
+   TBD: Should the ErlEF offer to help directly? If yes: outline: contact details
```

Comment:
To discuss

Reply:
* Generic Contact method to find someone to get in touch with
* Do not send disclosures!